seed/main/hi_Latn.xml

```diff
@@ -631,7 +631,6 @@ annotations.
 			<script type="Mero">Meroitic</script>
 			<script type="Mlym">Malayalam</script>
 			<script type="Mong">Mongolian</script>
-			<script type="Moon">Moon</script>
```

Comment:
Thus should not be changed in the data (and besides I have moved this file to common in a different PR in progress). There's a  list somewhere of codes whose display names can legitimately match the code, that should be updated.

Reply:
@pedberg-icu  the issue here is that this exact element is duplicated, with the same content.

Reply:
look at like 636/635 of this same file.